### ISSUE TITLE
Question re: pretrained model

### ISSUE THREAD

**Issue by adamklein** [AntonMu/TrainYourOwnYOLO#27]
Hi, thanks for the demo! For the pre-trained cat face detector model included in this repository, did you train it with only the 100 annotated examples included? (because the performance is pretty good already with so little data - any stats on the accuracy you achieved?).

**Comment by AntonMu** [comment]
Hi Adam, you are welcome. Yes, this model was only trained on the labeled examples included. 

But, of course, by using transfer learning it also uses the knowledge from the original darknet weights that were trained for a much more general task. 

I don't have any stats on accuracy and any number would depend on the metric used. One avenue would be to create test images and measure Intersection over Union or do some scoring using the YOLOv3 loss function itself. In either case, one would need to label some test images which I haven't done. So far the only metric is the eye-ball metric ;)

**Comment by adamklein** [comment]
Well I can confirm it works well on out-of-sample data. Thanks again!!
